bioinformatics_template_markdown.py

```diff
@@ -1,116 +0,0 @@
-#!/usr/bin/env python
-####
-# Monash Bioinformatics Platform Template Script 1.2
-####
-#   steve.androulakis@monash.edu
-####
-## Note: Set template_dir in run()!
-
-from __future__ import print_function
-import sys
-import markdown
-import subprocess
-from jinja2 import Environment, FileSystemLoader
-
-
-def warning(*objs):
-    print("WARNING: ", *objs, file=sys.stderr)
-
-
-def render_markdown_content(filename):
-    with open(filename, "r") as myfile:
-        data = myfile.read()
-
-    data = markdown.markdown(data,\
-        extensions=['markdown.extensions.fenced_code'])
-    return data
-
-
-def process_r_markdown(rmarkdownfile):
-    command = "Rscript -e \'require(knitr);" +\
-    "opts_knit$set(out.format = \"md\"); knit(\"%s\")\'" % (rmarkdownfile)
-
-    warning(subprocess.Popen(command, shell=True,
-        stdout=subprocess.PIPE).stdout.read())
-
-    filename = rmarkdownfile.rsplit('.')[0].rsplit('/')[-1]
-    filename = "%s.md" % filename
-
-    return filename
-
-
-def run():
-    ####
-    # Monash Bioinformatics Platform Template Script 1.2
-    ####
-    #   steve.androulakis@monash.edu
-    ####
-    template_dir = '/software/apps/report_template_script/templates'
```

Comment:
This is an issue I created :) But if you want to turn this into a nice package, remove the hard-coding of this location!!